src/Attachment.tsx

```diff
@@ -33,6 +33,8 @@ const queryString = (query: QueryParams) =>
     .map(key => encodeURIComponent(key) + '=' + encodeURIComponent(query[key].toString()))
     .join('&');
 
+const isNil = (value: any) => value === null || typeof value === "undefined"
```

Comment:
lodash has a `_.isNil` function which returns null when the value is nullish. I took the name from there (altough the lodash implementation uses non-strict == comparison to null, but the result should be the same)